identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/screens/MainScreen.kt

```diff
@@ -53,13 +74,24 @@ fun MainScreen(sampleName: String, application: Application) {
                     .fillMaxSize()
                     .padding(it)
             ) {
-                // composable function that wraps the MapView
-                ComposeMapView(
+                MapView(
                     modifier = Modifier
                         .fillMaxSize()
                         .weight(1f)
                         .animateContentSize(),
-                    mapViewModel = mapViewModel
+                    arcGISMap = mapViewModel.arcGISMap,
+                    viewpointOperation = MapViewpointOperation.Set(viewpoint = viewpoint),
+                    mapViewProxy = mapViewProxy,
+                    onSingleTapConfirmed = { singleTapConfirmedEvent ->
+                        sampleCoroutineScope.launch {
+                            val identifyResult = mapViewProxy.identifyLayers(
```

Comment:
A comment here about the `identifyLayers` function would be helpful:
```suggestion
                            // identify the layers on the tapped coordinate
                            val identifyResult = mapViewProxy.identifyLayers(
```